### PR TITLE
Added RedM

### PR DESCRIPTION
**Pull request by tr1ckz**
Added RedM support

### CODE REVIEW

**Review by lze3 — APPROVED** [review]

**Review by PLOKMJNB — APPROVED** [review]

File: scripts/cl_ui.lua [hunk 1]
```diff
@@ -1,3 +1,14 @@
+local isRDR = not TerraingridActivate and true or false
+local dismissKey
+local dismissKeyGroup
+if isRDR then 
+    dismissKey = 0xD9D0E1C0
+    dismissKeyGroup = 1
+else
+    dismissKey = 22
+    dismissKeyGroup = 0
+end
```
Comment (lze3) [inline scripts/cl_ui.lua]:
```suggestion
local dismissKey = isRDR and 0xD9D0E1C0 or 22
local dismissKeyGroup = isRDR and 1 or 0
```

**Review by lze3 — COMMENTED** [review]
Just to clean up code a bit.

**Comment by tabarra** [comment]
Now we need to update `cl_logger.lua` to work with RedM, or simply disable it for now...  
Anybody up for the challenge?

**Comment by IqFiskMos** [comment]
disable it for redm for now and when we have a fix for it enable again.

**Pull request merged**